docs/action_api.md

```diff
@@ -55,39 +56,40 @@ Here's a full description of the available options for the `ActionList` response
 
 ```ts
 interface ActionList {
+  /** A label for the Hub that will be shown in the Looker admin page **/
+  label: string
   /** (required) An array of action definition objects. */
   integrations: ActionDefinition[]
 }
 
 interface ActionDefinition {
   /** (required) A unique name for the action. This should be unique across all actions in the hub. */
   name: string
-  /** (required) An absolute URL of the execute endpoint for this action. */
-  url: string
   /** (required) A human-readable label for the action. */
   label: string
-  /** (required) A list of action types the action supports. Valid values are: "cell", "query", "dashboard". */
-  supported_action_types: string[]
-
-  /** An absolute URL of the form endpoint for this action. */
-  form_url?: string
   /** Description of the action. */
   description?: string
+  /** A Data URI reprensenting an icon image for the action. */
+  icon_data_uri?: string
+  /** (required) An absolute URL of the execute endpoint for this action. */
+  url: string
+  /** An absolute URL of the form endpoint for this action. */
+  form_url?: string
   /** Array of params for the action. */
   params?: Param[]
+  /** A list of descriptions of required fields that this action is compatible with. If there are multiple entries in this list, the action requires more than one field. */
```

Comment:
Not sure if you want to add the `user_attribute_params` info here? https://docs.looker.com/sharing-and-publishing/action-hub#adding_user_attributes_to_custom_actions

Reply:
good point - done!